restorer_test.go

```diff
@@ -289,157 +283,200 @@ func testRestorer(t *testing.T, when spec.G, it spec.S) {
 						want := "echo text from cache-only layer\n"
 						h.AssertEq(t, string(got), want)
 					})
-				})
-			})
 
-			when("there is a cache=false layer", func() {
-				var meta string
-				it.Before(func() {
-					meta = "[metadata]\n  cache-false-key = \"cache-false-val\""
-					h.AssertNil(t, writeLayer(layersDir, "buildpack.id", "cache-false", meta, cacheFalseLayerSHA))
-					h.AssertNil(t, restorer.Restore(testCache))
+					when("buildpack API < 0.6", func() {
```

Comment:
why do we need this `when` if we have the `SkipIf`?

Reply:
Was removed as part of [this commit](https://github.com/buildpacks/lifecycle/pull/592/commits/77b09a929e3ba31c0f01d3186696b101dffb5e72).